atd-etl/afd_ems_import/docker-compose.yml

```diff
@@ -6,6 +6,4 @@ services:
     volumes:
       - .:/app
     env_file:
-      - ./env
-    environment:
-      - ENVIRONMENT=development
```

Comment:
I removed this since we already default to `development` in the script, and we have an `.env` file. I missed this and kept running on development after changing my `.env` to production to do the backfill. 🙃